spec/Section 2 -- Language.md

```diff
@@ -779,6 +779,25 @@ curly-braces `{ }`.  The values of an object literal may be any input value
 literal or variable (ex. `{ name: "Hello world", score: 1.0 }`). We refer to
 literal representation of input objects as "object literals."
 
+**Input object fields are unordered**
```

Comment:
This is just added clarification - this section matches a similar section for Field Arguments.